src/devices/Bmx280/samples/Bme280.sample.cs

```diff
@@ -23,53 +23,70 @@ static async Task Main(string[] args)
             //set this to the current sea level pressure in the area for correct altitude readings
             const double defaultSeaLevelPressure = 1033.00;
 
-            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress);
+            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress + 1);
             var i2cDevice = new UnixI2cDevice(i2cSettings);
             var i2CBmpe80 = new Bme280(i2cDevice);
 
             using (i2CBmpe80)
             {
                 while (true)
                 {
-                    ////set mode forced so device sleeps after read
+                    Console.WriteLine("Let's get this party started!");
```

Comment:
NIT: Nice! but we should probably remove it :)

Reply:
I did that mainly so I could see what loop we were on (I was using the sample app to test with). I'll pull it out.